tests/strings_ut/strings_ut.c

```diff
@@ -134,7 +134,7 @@ BEGIN_TEST_SUITE(TEST_SUITE_NAME_FROM_CMAKE)
         count = umock_c_negative_tests_call_count();
         for (index = 0; index < count; index++)
         {
-            char tmp_msg[64];
+            char tmp_msg[128];

```

Comment:
>128 [](http://example.com/codeflow?start=25&length=3)

/consider

You might want to add a define for this so you don't have to go through all them again if for some chance.

Reply:
Good point, will do.